app/backend/src/couchers/models.py

```diff
@@ -239,6 +241,8 @@ class User(Base):
     # e.g. cus_JjoXHttuZopv0t
     stripe_customer_id = Column(String, nullable=True)
 
+    new_notifications_enabled = Column(Boolean, nullable=False, server_default=text("false"))
```

Comment:
> How is false here different from having nothing in the notification_preferences table?

I've decided to make the notification preferences overrides over the default config. So that way there is some default that we can change over time, so for example if there is a new notification type then existing users will get notifications for it.

The other thing is that there should be some security notifications that the user shouldn't be able to turn off (e.g. password changed). Similarly I've been thinking something along the lines of; if a user has said they're hosting last minute, they have to have some type of notification enabled for new host requests.

Anyway, this is just a temporary thing until we phase out emails and switch everyone to thsi.